src/core/manager.c

```diff
@@ -960,7 +957,7 @@ int manager_new(RuntimeScope runtime_scope, ManagerTestRunFlags test_run_flags,
         }
 
         /* Reboot immediately if the user hits C-A-D more often than 7x per 2s */
-        m->ctrl_alt_del_ratelimit = (const RateLimit) { .interval = 2 * USEC_PER_SEC, .burst = 7 };
+        m->ctrl_alt_del_ratelimit = (RateLimit) { .interval = 2 * USEC_PER_SEC, .burst = 7 };
```

Comment:
why drop the const?

Reply:
Why `const` is necessary?

Reply:
The same reason as https://github.com/systemd/systemd/commit/9bb64f1b8d2252d558f209a04b171d65b5ef131b I reckon.